<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pseDialog</class>
 <widget class="QDialog" name="pseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pulse Sequence Editor</string>
  </property>
  <widget class="QGroupBox" name="gbCurrentPoint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Current time point: 1 of 1</string>
   </property>
   <widget class="QLineEdit" name="leName">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Defines the name of this time point.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of clocks from start of sequence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>276</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Time from start of sequence (not used)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leClocks">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTime">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Not currently used</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbDigitalOut">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>235</y>
     <width>131</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Digital out</string>
   </property>
   <widget class="QCheckBox" name="chkDOA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOG">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOH">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOJ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOO">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDOI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDON">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbDCbias">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>235</y>
     <width>296</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>DC bias outputs</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>45</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB11">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB12">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB16">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>320</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB13">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>325</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>125</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 11</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB14">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>85</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>165</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB9">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB10">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>285</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 15</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>245</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 14</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDCB15">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>205</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CH 13</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbNext">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Advance to next time point</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbInsert">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Insert a time point after this one</string>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDelete">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete this time point</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrevious">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>130</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Go back to previous time point</string>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frmLoop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>174</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>11</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of cycles</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCycles">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>15</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of times to loop</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboLoop">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>12</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Point to loop back to</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkLoop">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>11</y>
      <width>106</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Check this to cause this time point to loop back to the labeled point</string>
    </property>
    <property name="text">
     <string>Loop to</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbInsertPulse">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>130</y>
     <width>105</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Insert a time point after this one</string>
   </property>
   <property name="text">
    <string>Insert pulse</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbInsertPulse">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>162</y>
     <width>441</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Insert pulse parameters</string>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>225</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramp up number of steps</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>295</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramp down number of steps</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>335</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramp step size</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePulseWidth">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>225</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRampUp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRampDwn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>305</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRampStepSize">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>230</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Clock cycles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>340</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Clock cycles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbInsertRamp">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>385</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate pulse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>396</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>This function will generate the time points needed to make a voltage pulse on the DC bias channel number selected. If you are inserting this between existing time points in a sequence then there must be adequate space or an error will be reported.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DC bias channel </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>145</y>
      <width>141</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse voltage, note the currrect time point value will define the resting voltage</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePulseChannel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePulseVoltage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>165</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>170</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbInsertCancel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>386</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <zorder>label_21</zorder>
   <zorder>label_22</zorder>
   <zorder>label_23</zorder>
   <zorder>label_24</zorder>
   <zorder>lePulseWidth</zorder>
   <zorder>leRampUp</zorder>
   <zorder>leRampDwn</zorder>
   <zorder>leRampStepSize</zorder>
   <zorder>label_25</zorder>
   <zorder>label_26</zorder>
   <zorder>pbInsertRamp</zorder>
   <zorder>label_28</zorder>
   <zorder>label_29</zorder>
   <zorder>lePulseChannel</zorder>
   <zorder>lePulseVoltage</zorder>
   <zorder>label_30</zorder>
   <zorder>label_27</zorder>
   <zorder>pbInsertCancel</zorder>
  </widget>
  <zorder>gbDigitalOut</zorder>
  <zorder>gbDCbias</zorder>
  <zorder>gbCurrentPoint</zorder>
  <zorder>pbNext</zorder>
  <zorder>pbInsert</zorder>
  <zorder>pbDelete</zorder>
  <zorder>pbPrevious</zorder>
  <zorder>frmLoop</zorder>
  <zorder>pbInsertPulse</zorder>
  <zorder>gbInsertPulse</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
